foundations/html_css/html-foundations/links-and-images.md

```diff
@@ -260,15 +260,22 @@ Go ahead and update the `odin-links-and-images` project with width and height ta
 
 This section contains questions for you to check your understanding of this lesson on your own. If you’re having trouble answering a question, click it and review the material it links to.
 
-- [What element is used to create a link?](#anchor-elements)
-- [What is an attribute?](#attribute)
-- [What attribute tells links where to go to?](#where-to-go)
-- [What security considerations must be taken if you wish to use the target attribute to open links in a new tab/window?](#target-security)
-- [What is the difference between an absolute and relative link?](#absolute-and-relative-links)
-- [Which element is used to display an image?](#images)
-- [What two attributes do images always need to have?](#two-attributes)
-- [How do you access a parent directory in a filepath?](#parent-filepath)
-- [What are the four main image formats that you can use for images on the web?](https://internetingishard.netlify.app/html-and-css/links-and-images/#image-formats)
+- [Introduction](#introduction)
+- [Lesson overview](#lesson-overview)
+- [Preparation](#preparation)
+- [Anchor elements](#anchor-elements)
+- [Opening links in a new tab](#opening-links-in-a-new-tab)
+- [Absolute and relative links](#absolute-and-relative-links)
+  - [Absolute links](#absolute-links)
+  - [Relative links](#relative-links)
+  - [A metaphor](#a-metaphor)
+- [Images](#images)
+- [Parent directories](#parent-directories)
+- [Alt attribute](#alt-attribute)
+- [Image size attributes](#image-size-attributes)
+- [Assignment](#assignment)
+- [Knowledge check](#knowledge-check)
+- [Additional resources](#additional-resources)
```

Comment:
```suggestion
- [What element is used to create a link?](#anchor-elements)
- [What is an attribute?](#attribute)
- [What attribute tells links where to go to?](#where-to-go)
- [What security considerations must be taken if you wish to use the target attribute to open links in a new tab/window?](#target-security)
- [What is the difference between an absolute and relative link?](#absolute-and-relative-links)
- [Which element is used to display an image?](#images)
- [What two attributes do images always need to have?](#two-attributes)
- [How do you access a parent directory in a filepath?](#parent-filepath)
- [What are the four main image formats that you can use for images on the web?](https://internetingishard.netlify.app/html-and-css/links-and-images/#image-formats)
```
We need to put these back to the way they were. Let me know if you have any questions about this

Reply:
yeah no problem. I have no idea how that changed to begin with. may I ask what would be the best way to handle this case on github. would it be amend and force with lease, add another commit and squash or straight up add a new commit and push it?

Reply:
The easiest thing for you would just be to add a new commit with that change and push that. I'll be able to squash your commits on my end when I merge your branch, so you don't have to worry about dealing with that.

Reply:
something is very odd. as suspected I never made this change to begin with and whenever I manually remove it and copy back the original content it changes back automatically whenever I save the file. I even tried resolving conflicts and replace the changes and it still resets  whenever I save even while resolving the conflicts. this happens whether I'm on this branch or on the main branch. any idea what this could be? and would it be better to close the pull request, delete the branch and redo it all?

Reply:
should be good now. apologies for the delay

Reply:
It's no problem. Merging now :tada: